tests/test_pivot.py

```diff
@@ -136,16 +137,220 @@ def test_pivot_longer_rep_index_groups(self):
             stang,
             index_to = "idx",
             columns=["E_00","mu_00","E_45","mu_45","E_90","mu_90"],
-            names_to="var",values_to="val"
+            names_to="var",
+            values_to="val"
+        )
+
+        long_index = list(long["idx"])
+        mod = max(long_index)
+        single_index = list(range(0, mod))
+
+        if single_index == long_index[0:mod]:
+            result = True
+        else:
+            result = False
+
+        self.assertTrue(result)
+
+    def test_pivot_longer_select(self):
+        """ Test if pivot_longer is compatible with gr.tran_select or any input
+            of a DataFrame instead as 'columns' argument
+        """
+        stang = data.df_stang_wide
+        long = gr.tran_pivot_longer(
+            stang,
+            columns=(gr.tran_select(stang,gr.matches("\\d+"))),
```

Comment:
This implies that the `columns` argument is not correctly implemented; the call signature should be

```
gr.tran_pivot_longer(
    stang,
    columns=gr.matches("\\d+"),
    names_to="var",
    values_to="val",
)
```

Compare this with the existing `gr.tran_select()` usage

```
gr.tran_select(stang, gr.matches("\\d+"))
```